src/workers/local.ts

```diff
@@ -0,0 +1,12 @@
+import { processSubscriptionsHandler } from '../handlers';
+import { WorkerParameters } from '../types';
+
+export const spawn = async ({ providerSponsorSubscriptions, baseLogOptions }: WorkerParameters): Promise<string> =>
+  new Promise((resolve, reject) => {
+    processSubscriptionsHandler({ providerSponsorSubscriptions, baseLogOptions }).then((data) => {
+      if (!data.ok) {
+        reject(data.message);
+      }
+      resolve(data.message);
+    });
+  });
```

Comment:
Do we even need this file and the local `spawn` version? Why don't we just call the `processSubscriptionsHandler` directly for the `local` case?

Reply:
You're right, we can do it that way as well :+1: